### ISSUE TITLE
Manually triggering the creation of a component

### ISSUE THREAD

**Issue by jeromegn** [x-tag/core#110]
So, my component is finally ready. I've been able to test it correctly in a controlled environment.

As I'm adding it to a Rails project which uses Turbolinks (similar to PJAX), it seems like my custom component is not being created anymore. On the first page load, it's all good, but on subsequent loads via turbolinks, the component is not created.

Therefore, I get the bland element. None of the lifecycle callbacks are fired. I get no errors in the console, so I imagine it's just a question of triggering that creation process.

Does x-tag fire a special event for this to happen? If so, I couldn't find it in my search.

Basically: what triggers the creation of a component?
